docs/4.3/architecture/authentication.md

```diff
@@ -226,9 +226,9 @@ back-ends as shown in the table below:
 
 Data Type        | Supported Back-ends       | Notes
 -----------------|---------------------------|---------
-Cluster state    | `dir`, `etcd`, `dynamodb` | Multi-server (HA) configuration is only supported using `etcd` and `dynamodb` back-ends.
-Audit Log Events | `dir`, `dynamodb`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
-Recorded Sessions| `dir`, `s3`               | `s3` is mandatory if `dynamodb` is used for the audit log.
+Cluster state    | `dir`, `etcd`, `dynamodb`,`firestore` | Multi-server (HA) configuration is only supported using `etcd`, `dynamodb`, and `firestore` back-ends.
+Audit Log Events | `dir`, `dynamodb`, `firestore`         | If `dynamodb` is used for the audit log events, `s3` back-end **must** be used for the recorded sessions.
```

Comment:
I think I asked this on the other PR, but do we have a similar requirement that you **must** use Google storage for recorded sessions if you use `firestore` for audit logs? I have to confess that I don't completely understand why we have this hard requirement for `dynamodb` either.

Reply:
That's a good question. I don't think so.. when I was trying it out a while ago but I'll make a note to double check. I think the reason for using dynamodb/s3 is for HA and keeping it consistent but I can see a reason to still keep them on disk.